.github/workflows/build.yml

```diff
@@ -1,8 +1,6 @@
 name: build
 
-on:
-  pull_request:
-    types: [opened, edited, synchronize, reopened]
+on: [pull_request]
```

Comment:
but will it work on direct pushes?

Reply:
No. There is no way to specify push type accroding to https://help.github.com/en/github/automating-your-workflow-with-github-actions/events-that-trigger-workflows#push-event-push

So, either we will have no CI for direct push, or no CI for PRs from forks. Or there always would be 2 runs in case of PRs: 1 for push and 1 for PR.

Reply:
Ok, will do PRs then :)